.travis.yml

```diff
@@ -18,22 +18,24 @@ services:
 
 matrix:
   include:
-    - php: 5.3
+    - php: 5.4
       env: COMPOSER_FLAGS="--prefer-lowest"
```

Comment:
@alexislefebvre my goal is to make sure that we simply bump the lowest version to a sane point .. f.e. I just bumped the minimal version to 2.3.2 since it makes no sense to maintain a 2.3.1 specific hack